README.md

```diff
@@ -7,26 +7,20 @@ This addon provides an API for authoring a [Router service](https://api.emberjs.
 
 
 ## Compatibility
-------------------------------------------------------------------------------
-
-* Ember.js v3.12 or above
-* Ember CLI v3.24 or above
-* Node.js v14 or above
 
+- Ember.js v3.12 or above
+- Embroider or ember-auto-import v2
```

Comment:
Could you please link the embroider blueprint to the PR description for better testing and review @SergeAstapov ?

Reply:
@villander this was already done